src/command/run.rs

```diff
@@ -50,6 +51,40 @@ pub fn run(target: Target, in_file: PathBuf) -> Result<(), String> {
                 .write_all(&s)
                 .map_err(|e| format!("Could not write to stdout: {}", e))?;
         }
+        Target::Qbe => {
+            let dir_path = "./"; // TODO: Use this for changind build directory
+            let filename = in_file.file_stem().unwrap().to_str().unwrap(); 
+            let ssa_path = format!("{dir_path}{}.ssa", filename); 
+            let asm_path = format!("{dir_path}{}.s", filename); 
+            let exe_path = format!("{dir_path}{}.exe", filename);
+
+            let mut ssa_file = OpenOptions::new()
+                        .read(true)
+                        .write(true)
+                        .create(true)
+                        .open(&ssa_path).unwrap();
+            let buff = *buf;
+            ssa_file.write_all(&buff).unwrap();
+
+
+            // SSA to ASM
+            Command::new("qbe")
+                .arg(&ssa_path)
+                .arg("-o")
+                .arg(&asm_path)
+                .spawn().unwrap();
+
+            // ASM to EXE
+            Command::new("gcc")
+                .arg(&asm_path)
+                .arg("-o")
+                .arg(&exe_path)
+                .spawn().unwrap(); 
+
+            // Run the EXE
+            Command::new(exe_path)
+                .spawn().unwrap();
+        }
```

Comment:
This should be in a different PR as it's not a requirement for the shorthand returns

Reply:
I personally don't mind the addition here. Saves time and trouble. ✌️